### PR TITLE
add a function to remove unused imports

### PR DESCRIPTION
**Pull request by danielwuz**
I found a code snippet to remove unused imports in python code. I think it might be a nice thing to have in spacemacs.

### CODE REVIEW

File: contrib/lang/python/README.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ Test commands (start with <kbd>m t</kbd> or <kbd>m T</kbd>):
 <kbd>SPC m h H</kbd>  | open documentation in `firefox` using [pylookup][pylookup]
 <kbd>SPC m v</kbd>    | activate a virtual environment with [pyenv][pyenv]
 <kbd>SPC m V</kbd>    | activate a virtual environment with [pyvenv][pyvenv]
+<kbd>SPC c i</kbd>    | remove unused imports with [autoflake][autoflake]
```
Comment (cpaulik) [inline contrib/lang/python/README.md]: This is a typo, should be `<kbd>SPC m c i</kbd>`

**Comment by syl20bnr** [comment]
Great ! :+1: 
I improved it a bit to add a check the presence of autoflake and added some doc about autoflake. I also moved the key binding to the `SPC m r` prefix for refactoring.
Related commit: https://github.com/syl20bnr/spacemacs/commit/c3d7635

Thank you !
Cherry-picked into `develop` branch, you can safely delete your branch.

File: contrib/lang/python/README.md (hunk 1 above)
Comment (syl20bnr) [inline contrib/lang/python/README.md]: I fixed it, thank you.

**Comment by danielwuz** [comment]
Thank you all!

**Comment by CestDiego** [comment]
as there are lot's of packages needed, can we have a requirements.pip file? :P

**Comment by syl20bnr** [comment]
Could be a good idea.

-syl20bnr-

**Pull request closed without merging**